### PR TITLE
Add ability to specify tenant ID through env variable

### PR DESCRIPTION
**Pull request by aallawala**
#### What this PR does
This PR allows for the `load-testing-with-k6.js` K6 job to set the tenant_name. This enables writing to clusters that have authentication enabled, requiring a tenant name to be set for all writes.

#### Which issue(s) this PR fixes or relates to
No issue present at the moment. Please let me know if I should create an associated issue for this.
Fixes #<issue number>

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by 56quarters — APPROVED** [review]
These changes LGTM, thanks! Please just sign the CLA and we'll get this merged.

File: operations/k6/load-testing-with-k6.js [hunk 1]
```diff
@@ -89,19 +89,30 @@ const HA_REPLICAS = parseInt(__ENV.K6_HA_REPLICAS || 1);
  * @constant {number}
  */
 const HA_CLUSTERS = parseInt(__ENV.K6_HA_CLUSTERS || 1);
+/**
+ * Allow for this script to write to a specific tenant in the Mimir cluster.
+ * By default, no tenant is specified requiring the cluster to have multi-tenancy disabled.
+ * @constant {string}
+*/
+const TENANT_NAME = __ENV.K6_TENANT_NAME || '';
```
Comment (pracucci) [inline operations/k6/load-testing-with-k6.js:97]: Do you mind calling it `TENANT_ID` to keep it consistent with Mimir, please?

Comment (aallawala) [reply operations/k6/load-testing-with-k6.js:97]: Thanks for taking a look. No, I don't mind. Will push something in a moment

**Review by pracucci — APPROVED** [review]
LGTM! @aallawala please sign the CLA, otherwise we can't merge this PR. Thanks!

**Pull request merged**